examples/greeter-extension/src/__tests__/greeter-extension.acceptance.ts

```diff
@@ -0,0 +1,80 @@
+// Copyright IBM Corp. 2019. All Rights Reserved.
+// Node module: @loopback/example-greeter-extension
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {Application, bind, createBindingFromClass} from '@loopback/core';
+import {expect} from '@loopback/testlab';
+import chalk from 'chalk';
+import {
+  asGreeter,
+  Greeter,
+  GreeterComponent,
+  GreetingService,
+  GREETING_SERVICE,
+} from '..';
+
+describe('greeter-extension-pont', () => {
+  let app: Application;
+  let greetingService: GreetingService;
+
+  beforeEach(givenAppWithGreeterComponent);
+  beforeEach(findGreetingService);
+
+  it('greets by language', async () => {
+    let msg = await greetingService.greet('en', 'Raymond');
+    expect(msg).to.eql('Hello, Raymond');
+    msg = await greetingService.greet('zh', 'Raymond');
+    expect(msg).to.eql('Raymond，你好！');
+  });
+
+  it('supports options for the extension point', async () => {
+    // Configure the extension point
+    app.bind('services.GreetingService.options').to({color: 'blue'});
+    greetingService = await app.get(GREETING_SERVICE);
+    const msg = await greetingService.greet('en', 'Raymond');
+    expect(msg).to.eql(chalk.keyword('blue')('Hello, Raymond'));
```

Comment:
if we change 
> expect(msg).to.eql(chalk.keyword('blue')('Hello, Raymond'));
to
```
expect(msg).to.eql(chalk.keyword('green')('Hello, Raymond'));
```

the test still passes since the `msg` string variable is still 'Hello, Raymond' and doesn't contain any `chalk` notation.

The usage of chalk is usually within a console.log() statement : 
```
console.log(chalk.blue('Hello') + ' World' + chalk.red('!'));
```
In a command window, we can see the effect of `chalk` colours.

So perhaps this test should check that the greeter extension point actually had the options set on it, that we requested?

Reply:
Which OS do you test? I see a failure as follows:
```
AssertionError: expected '\u001b[38;5;21mHello, Raymond\u001b[39m' to equal '\u001b[38;5;34mHello, Raymond\u001b[39m'
      + expected - actual
```

Reply:
I'll add a test for `greetingService.options` anyway.